xskillscore/tests/test_deterministic.py

```diff
@@ -212,25 +301,93 @@ def test_correlation_metrics_xr_dask(
 @pytest.mark.parametrize("metrics", distance_metrics)
 @pytest.mark.parametrize("dim", AXES)
 @pytest.mark.parametrize("weight_bool", [True, False])
-@pytest.mark.parametrize("skipna", [True, False])
-@pytest.mark.parametrize("has_nan", [True, False])
 def test_distance_metrics_xr_dask(
-    a_dask, b_dask, dim, weight_bool, weights_dask, metrics, skipna, has_nan
+    a_dask, b_dask, dim, weight_bool, weights_dask, metrics
 ):
     """Test whether distance metrics for xarray functions can be lazy when
-    chunked by using dask and give same results."""
+    chunked by using dask and give same results as np array."""
```

Comment:
Yes the test name was misleading. We don't really check whether result is also chunked, as we often do when calling the function _dask